viewmaster/link.py

```
@@ -22,36 +37,95 @@
+    link_logger_name = LOGNAME.replace('viewmaster', 'link')
+    logger = pdslogger.PdsLogger(link_logger_name, limits={'info': -1, 'normal': -1},
+                                 pid=True)
+
+    LOG_FILE = LOG_ROOT_PREFIX_ + 'link.log'
+    info_logfile = os.path.abspath(LOG_FILE)
+
+    try:
+        info_handler = pdslogger.file_handler(info_logfile, level=logging.INFO,
+                                              rotation='midnight')
+        logger.add_handler(info_handler)
+    except (OSError, IOError) as e:
```

Ruff (0.14.10)

Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)